resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,171 @@
+# Troubleshooting Campaign push notifications
+                         
+## To set up server key/APNS server certificate in the Campaign instance, complete the following steps:  
+
+{% tabs %}
+
+{% tab title="Android" %}
+### Set up a server key
+
+   1. To configure the app in Google Firebase, see [Firebase Console](https://console.firebase.google.com/) and sign in using your Google Developer credentials. 
+   2. Add a project and add your app under that project using the application ID that you used in the app.  
+   For more information, see [Add Firebase to your Android project](https://firebase.google.com/docs/android/setup#console).
+   3. Add the generated **“google-service.json”** to the project and set up the project and module level gradle files.  
+   For more information, see [Add Firebase to your Android project](https://firebase.google.com/docs/android/setup#console).
+   4. Verify that Firebase server key is configured in the Campaign instance by clicking **_Adobe Campaign -> Administrator -> Channels -> Mobile App (AEP SDK)_**.    .
+   5. Select the mobile app from the list and verify that the push channel settings status for Android is _Android key set up success_.  
+   
+   ![Server key configuration in Android](../../.gitbook/assets/android_server_key.png "Server key configuration in Android.")
+{% endtab %}
+
+{% tab title="iOS" %}
+
+### Set up the APNS server certificate
+
+   1. Add Push Notification entitlement to your app.  
```

Comment:
These steps look like they are not flush left...please remove the spaces so that they appear flush left with the heading.